java/form/src/org/netbeans/modules/form/JavaCodeGenerator.java

```diff
@@ -2554,6 +2517,57 @@ private void generateListenerAddCode(RADComponent comp,
             generateCatchCode(exceptions, codeWriter);
     }
 
+    private void generateInnerClasses(Writer codeWriter, EventSetDescriptor eventSetDesc,List<Event> eventList) throws IOException {
+
+        if(formModel.getSettings().getGenerateListenersAsLambda() && eventSetDesc.getListenerMethods().length==1) {
+            generateWithLambda(codeWriter,eventSetDesc,eventList);
+        }else{
```

Comment:
I don't think this is correct. A button for example can have multiple event handlers for `actionPerformed`. This will only generate one and ignore all others if there are more.

You can only generate a handler call using a method reference if:
`eventList.size() == 1 && eventSetDesc.getListenerMethods().length == 1 && eventList.get(0).getEventHandlers.length == 1`

If there are `>1` handlers, you would actually have to use a lambda as the PR title promises :)

also please add spaces before `({` and after `)}` and `,` to make it consistent to the rest of the code, thanks.

Reply:
Corrected, I didn't know I can add multiple events. 
![image](https://github.com/apache/netbeans/assets/61804231/5bc11912-0cc8-4b59-bd69-e8706e35cff8)
I added spaces like there and seems be clear.

Reply:
I have doubts about the space before (evt) only

Reply:
> I have doubts about the space before (evt) only

sorry, I meant spaces in your code, not in generated code. 

your code looks like this:
```java
        }else if(eventList.get(0).getEventHandlers().length > 1)
        {
```
it should look like:
```java
        } else if (eventList.get(0).getEventHandlers().length > 1) {
```

we try to avoid mixing too many styles in the same file, otherwise it looks like chaos.

Braces are usually on the same line and add spaces before `({` and after `)}` and `,`
```java
            generateWithLambda(codeWriter,eventSetDesc,eventList);
```
->
```java
            generateWithLambda(codeWriter, eventSetDesc, eventList);
```
thanks

Reply:
Formatting corrected